subt/ros/robot/launch/freyja.launch

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <launch>
   <node name="$(arg robot_name)_proxy" pkg="proxy" type="ros_proxy" args="$(arg robot_name)">
-    <remap from="$(arg robot_name)/front_scan" to="$(arg robot_name)/scan_front"/>
-    <remap from="$(arg robot_name)/image_raw/compressed" to="$(arg robot_name)/rgbd_front/image_raw/compressed"/>
-    <remap from="$(arg robot_name)/front/depth" to="$(arg robot_name)/rgbd_front/depth"/>
-    <remap from="$(arg robot_name)/points" to="$(arg robot_name)/rgbd_front/points"/>
+    <remap from="$(arg robot_name)/front_scan" to="$(arg robot_name)/nonexistent_scan_front"/>
+    <remap from="$(arg robot_name)/image_raw/compressed" to="$(arg robot_name)/nonexistent_rgbd_front/image_raw/compressed"/>
+    <remap from="$(arg robot_name)/front/depth" to="$(arg robot_name)/nonexistent_rgbd_front/depth"/>
+    <remap from="$(arg robot_name)/points" to="$(arg robot_name)/nonexistent_rgbd_front/points"/>
```

Comment:
:+1: But why don't we also remove the corresponding code from ros_proxy_node?

Reply:
The rosproxy is used also by X4 and Explorer, but yes it is the next step.